docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -942,3 +942,88 @@ spec:
   purpose: Informational
   tags: compliance, hunting
   contributors: nabilschear
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Windows Firewall, Domain Profile enabled (Windows)
+  query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\DomainProfile\EnableFirewall' AND CAST(data as integer) = 1;
+  description: "Checks for the existence of a Group Policy that enables the Domain Profile for Windows Defender Firewall."
+  resolution: "Contact your IT administrator to ensure your Windows system is receiving a Group Policy that enables the Domain profile for Windows Defender Firewall."
+  platforms: Windows
+  tags: compliance, CIS, CIS9.1.1
+  platform: windows
+  contributors: defensivedepth
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Windows Firewall, Private Profile enabled (Windows)
+  query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\PrivateProfile\EnableFirewall' AND CAST(data as integer) = 1;
+  description: "Checks for the existence of a Group Policy that enables the Private Profile for Windows Defender Firewall."
+  resolution: "Contact your IT administrator to ensure your Windows system is receiving a Group Policy that enables the Private Profile for Windows Defender Firewall."
+  platforms: Windows
+  tags: compliance, CIS, CIS9.2.1
+  platform: windows
+  contributors: defensivedepth
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Windows Firewall, Public Profile enabled (Windows)
+  query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\PublicProfile\EnableFirewall' AND CAST(data as integer) = 1;
+  description: "Checks for the existence of a Group Policy that enables the Public Profile for Windows Defender Firewall."
```

```suggestion
  description: "Checks if firewall is enabled using the public profile. The public profile is used to designate public networks such as Wi-Fi hotspots at coffee shops, airports, and other locations."
```

Same simplification suggestion. Context is here: